integration-tests/src/tests/client/features/restrict_tla.rs

```diff
@@ -0,0 +1,59 @@
+use super::super::process_blocks::create_account;
+use crate::tests::client::utils::TestEnvNightshadeSetupExt;
+use near_chain::ChainGenesis;
+use near_chain_configs::Genesis;
+use near_client::test_utils::TestEnv;
+use near_primitives::errors::{ActionError, ActionErrorKind};
+use near_primitives::types::{AccountId, BlockHeight};
+use near_primitives::views::FinalExecutionStatus;
+use near_primitives_core::version::PROTOCOL_VERSION;
+use nearcore::config::GenesisExt;
+
+#[test]
+fn test_create_top_level_accounts() {
+    let epoch_length: BlockHeight = 5;
+    let account: AccountId = "test0".parse().unwrap();
+    let mut genesis = Genesis::test(vec![account.clone()], 1);
+    genesis.config.epoch_length = epoch_length;
+    genesis.config.protocol_version = PROTOCOL_VERSION;
```

Comment:
Hmm shouldn't that be nightly or the exact protocol version for this NEP, at least until this feature is stabilized?

Reply:
The test is only enabled for nightly. This is done in feature.rs with conditional compilation